flang/include/flang/Optimizer/Builder/Runtime/Command.h

```diff
@@ -0,0 +1,29 @@
+//===-- Command.h - generate assignment runtime API calls --------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef FORTRAN_OPTIMIZER_BUILDER_RUNTIME_COMMAND_H
+#define FORTRAN_OPTIMIZER_BUILDER_RUNTIME_COMMAND_H
+
+namespace mlir {
+class Value;
+class Location;
+} // namespace mlir
+
+namespace fir {
+class FirOpBuilder;
+}
+
+namespace fir::runtime {
+
+/// Generate a call to the `ArgumentCount` runtime.
+/// This calls the simple runtime entry point that then calls into the more
+/// complex function for calculating the count of command arguments.
```

Comment:
This makes it sound like getting the argument count is difficult and potentially time-consuming. `complex function for calculating the count of command arguments.` - aside from subtracting one and a comparison, it just returns the value of a store integer.

Reply:
Reduced comment size, following the style from Numeric.